herwigpp-toolfile.spec

```diff
@@ -11,12 +11,16 @@ cat << \EOF_TOOLFILE >%i/etc/scram.d/herwigpp.xml
 <tool name="herwigpp" version="@TOOL_VERSION@">
   <client>
     <environment name="HERWIGPP_BASE" default="@TOOL_ROOT@"/>
-    <environment name="LIBDIR" default="$HERWIGPP_BASE/lib"/>
-    <environment name="INCLUDE" default="$HERWIGPP_BASE/include"/>
+    <environment name="LIBDIR" default="$HERWIGPP_BASE/lib/Herwig7"/>
+    <environment name="INCLUDE" default="$HERWIGPP_BASE/include/Herwig7"/>
```

Comment:
@smuzaffar I am sorry it seems that I uploaded not the newest version of the tool file.

All parts with Herwig7 have to be changed to just Herwig as we found out in the old PR.
<environment name="LIBDIR" default="$HERWIGPP_BASE/lib/Herwig"/>
<environment name="INCLUDE" default="$HERWIGPP_BASE/include/Herwig"/>
<runtime name="HERWIGPATH" value="$HERWIGPP_BASE/share/Herwig"/>
Sorry for the additional efforts required. If you want and have time this week I will buy you a beer at the CERN R1.